ar/src/format/text.cpp

```diff
@@ -142,6 +144,49 @@ void TextFormatter::format(std::ostream& o, GlobalVariable* gv) const {
   }
 }
 
+void TextFormatter::format_header(std::ostream& o, const Function* f,
```

Comment:
Could we reuse this in `format(std::ostream&, const Function*, Namer&)` (below) so it's not duplicate code?

Reply:
Yes. Done in next commit.